x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/data_quality_panel/tabs/callouts/incompatible_callout/index.test.tsx

```diff
@@ -41,10 +39,10 @@ describe('IncompatibleCallout', () => {
       <TestProviders>
         <IncompatibleCallout
           enrichedFieldMetadata={[
-            eventCategoryWithWIldcard, // isEcsCompliant: false, indexInvalidValues.length: 0, isInSameFamily: true, `wildcard` and `keyword` are in the same family
-            eventCategoryWithUnallowedValues, // isEcsCompliant: false, indexInvalidValues.length: 2, isInSameFamily: true, `keyword` and `keyword` are in the same family
-            hostNameWithTextMapping, // isEcsCompliant: false, indexInvalidValues.length: 0, isInSameFamily: false, `keyword` and `text` are not in the family
-            sourceIpWithTextMapping, // isEcsCompliant: false, indexInvalidValues.length: 0, isInSameFamily: false, `ip` is not a member of any families
+            eventCategoryWithWIldcard, // `wildcard` and `keyword`
```

Comment:
super super nit: the I is accidentally capitalized I think `WIldcard` => `Wildcard`